.github/workflows/ci.yaml

```diff
@@ -153,38 +153,22 @@ jobs:
           git config --global user.name "GitHub Actions"
           git config --global user.email "actions@github.com"
 
-      # Run each test suite (tests and transcripts)
-      - name: unison-cli test
+      - name: test
         if: steps.cache-ucm-binaries.outputs.cache-hit != 'true'
-        run: stack build --fast --test unison-cli
-
-      - name: unison-core tests
-        if: steps.cache-ucm-binaries.outputs.cache-hit != 'true'
-        run: stack build --fast --test unison-core
-
-      - name: unison-parser-typechecker tests
-        if: steps.cache-ucm-binaries.outputs.cache-hit != 'true'
-        run: stack build --fast --test unison-parser-typechecker
-
-      - name: unison-sqlite tests
-        if: steps.cache-ucm-binaries.outputs.cache-hit != 'true'
-        run: stack build --fast --test unison-sqlite
-
-      - name: unison-syntax tests
-        if: steps.cache-ucm-binaries.outputs.cache-hit != 'true'
-        run: stack build --fast --test unison-syntax
-
-      - name: unison-util-bytes tests
-        if: steps.cache-ucm-binaries.outputs.cache-hit != 'true'
-        run: stack build --fast --test unison-util-bytes
-
-      - name: unison-util-cache tests
-        if: steps.cache-ucm-binaries.outputs.cache-hit != 'true'
-        run: stack build --fast --test unison-util-cache
-
-      - name: unison-util-relation tests
-        if: steps.cache-ucm-binaries.outputs.cache-hit != 'true'
-        run: stack build --fast --test unison-util-relation
+        run: |
+          for i in \
+            unison-cli \
+            unison-core \
+            unison-parser-typechecker \
+            unison-sqlite \
+            unison-syntax \
+            unison-util-bytes \
+            unison-util-cache \
+            unison-util-relation \
+
```

Comment:
this line intentionally left blank so that the packages above could uniformly end with ` \`